### PR TITLE
css-responsive-layout - Simon Kim.

### PR DESCRIPTION
**Pull request by Skim-329**
cssresponsive

### CODE REVIEW

File: css-responsive-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@media only screen and (min-width: 576px) {
+  .col-sm-half { width: 50%; }
+}
+
+@media only screen and (min-width: 768px) {
+  .col-lg-third { width: calc(100% / 3); }
+}
+
+@media only screen and (min-width: 768px) {
+  .col-lg-fourth { width: 25%; }
```
Comment (Zhiwutian) [inline css-responsive-layout/styles.css:43]: This selector and the one above should be in the same media query declaration block. You can put as much CSS you want in a media query block

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

**Comment by Skim-329** [comment]
cssresponsive

File: css-responsive-layout/styles.css (hunk 1 above)
Comment (Skim-329) [reply css-responsive-layout/styles.css:40]: Done!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**